app/components/navbar.hbs

```diff
@@ -0,0 +1,48 @@
+<nav class='nav'>
+  <button type='button' class='hamburger--toggle' {{on 'click' this.toggleNavbar}}>
+    <FaIcon @icon='bars' @size='2x' />
+  </button>
+
+  <ul class='nav__menu {{if this.isNavOpen "active" ""}}'>
+    <li>
+      <a href={{this.HOME_PAGE_URL}} class='nav__home'>
+        <img class='nav__home__img' src='assets/images/rds-logo.png' alt='Real_Dev_Squad' />
+      </a>
+    </li>
+    <li>
+      <a class='nav__home__link nav__element' href={{this.HOME_PAGE_URL}}>
+        Home
+      </a>
+    </li>
+    {{#each this.RDS_APPS_MAPPING as |siteURL|}}
+    <li><a class='nav__element' href={{siteURL.url}}>{{siteURL.siteName}}</a></li>
+    {{/each}}
+  </ul>
+
+  <div class='nav__login'>
+    {{#if @isLoggedIn }}
+    <div class='nav__user' role="button" class="user" {{on "click" this.toggleMenu}} {{on-click-outside
+      this.outsideClickMenu}}>
+      <div class='user__msg'>Hello, {{@firstName}}</div>
+      <img class='user__pic' src={{@profilePicture}} />
+      <Fa-Icon @icon='chevron-down' />
+    </div>
+    {{else}}
+    <a class='login' href={{this.AUTH_URL}}>
+      <button class='login__btn' type="button">
+        <span class='login__text1'>Sign In</span>
+        <span class='login__text2'>with GitHub</span>
```

Comment:
NIT: CSS naming could be different, text1 and text2 not descriptive

Reply:
Done